Makefile

```diff
@@ -17,7 +17,7 @@ ifndef VERBOSE
 MAKEFLAGS += --silent
 endif
 
-export NAMESPACE ?= devworkspace-controller
+export NAMESPACE ?= openshift-operators
```

Comment:
If it makes any sense we can stick to devworkspace-controller for development, but after chectl depends on it, I think it makes sense to unify default.

Reply:
marked as WIP since if openshift-operator is the default, then we must not remove that namespace on uninstall